src/components/BookItem.js

```diff
@@ -1,11 +1,27 @@
+import { PropTypes } from 'prop-types';
 import BookHeader from './BookHeader';
 import BookStatus from './BookStatus';
 
-const BookItem = () => (
-  <li className="book-item">
-    <BookHeader />
-    <BookStatus />
-  </li>
-);
+const BookItem = (props) => {
+  const { bookId, title, author } = props;
+
+  return (
+    <li className="book-item">
+      <BookHeader
+        bookId={bookId}
+        title={title}
+        author={author}
+      />
+      <p>{bookId}</p>
+      <BookStatus />
```

Comment:
Please implement `removeBookFromStore` button for deleting a book.

Reply:
This issue is well addressed! :clap: